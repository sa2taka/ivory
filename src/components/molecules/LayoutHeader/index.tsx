import React from 'react';
import './index.scss';
import { TitleLogo } from '@/components/atoms/TitleLogo';

export function LayoutHeader() {
  return (
    <header className="LayoutHeader h-16">
      <div className="h-full .z-0 flex relative items-center">
        <TitleLogo />
      </div>
    </header>
  );
}
