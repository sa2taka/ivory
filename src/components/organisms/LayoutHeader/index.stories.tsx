import React from 'react';
import { LayoutHeader as Component } from './index';
export default {
  title: 'organisms',
};
export const LayoutHeader = () => <Component />;
