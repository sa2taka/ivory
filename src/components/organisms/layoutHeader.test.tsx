import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LayoutHeader from './layoutHeader';

describe('LayoutHeader', () => {
  test('is rendered', () => {
    const wrapper = shallow(<LayoutHeader />);
    expect(wrapper).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
