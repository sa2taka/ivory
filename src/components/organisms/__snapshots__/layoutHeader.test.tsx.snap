// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutHeader is rendered 1`] = `
<header
  className="LayoutHeader h-16"
>
  <div
    className="h-full .z-0 flex relative items-center"
  >
    <div
      className="flex ml-4 items-center"
    >
      <img
        className="h-8 w-8"
        src="ivory-logo.png"
      />
      <span
        className="ml-2 text-2xl font-semibold"
      >
        Ivory
      </span>
    </div>
  </div>
</header>
`;
